### PR TITLE
Add hiding frame but preserving controls support

### PR DESCRIPTION
**Pull request by tete1030**
http://electron.atom.io/docs/api/frameless-window/#alternatives-on-macos
Added options to hide frame only but preserve window controls (“traffic lights”)

### CODE REVIEW

**Comment by nansthomas** [comment]
Hey ! I test your solution, but the windows is not draggable 🙁

**Comment by tete1030** [comment]
@nansthomas This Electron documentation describes how to make windows draggable: http://electron.atom.io/docs/api/frameless-window/#draggable-region

File: src/cli.js [hunk 1]
```diff
@@ -46,7 +46,8 @@ if (require.main === module) {
         .option('--inject <value>', 'path to a file to be injected', collect, [])
         .option('--full-screen', 'if the app should always be started in full screen')
         .option('--maximize', 'if the app should always be started maximized')
-        .option('--hide-window-frame', 'disable window frame and controls')
+        .option('--hide-window-frame', 'disable window frame and controls, should not be specified with --hide-title-bar together')
```
Comment (jiahaog) [inline src/cli.js]: This is specified but the option is unused, did you intend to use this as well?

**Comment by jiahaog** [comment]
I'm not sure if this is a complete solution to having a frameless window. Wouldn't it cause problems for users who set this flag and realise that they are unable to drag the window?

File: src/cli.js (hunk 1 above)
Comment (tete1030) [inline src/cli.js]: This option didn't added by me. It's from another pull request, and is used at https://github.com/tete1030/nativefier/blob/9bf593761a4727b43bf1f437fd4d5938e735158d/src/options/optionsMain.js#L63
The only thing I did was adding `--hide-title-bar` feature. But this feature conflicts with `--hide-window-frame` so I changed the description to notify user.
About the added line at Line 35 of `docs/api.md`, I think this line was missed by the person who added the `--hide-window-frame` feature, I just added missing part.

**Comment by ronjouch** [comment]
This PR was inadvertently closed due to us getting rid of the old `development` branch (we came back to merging on good'ol `master`). Changing base branch to `master` and reopening.

**Comment by ronjouch** [comment]
@tete1030 if you find the time to work on that, rebase welcome :)

**Pull request closed without merging**